### ISSUE TITLE
No script pics for error reports?

### ISSUE THREAD

**Issue by maryfries** [jmoenig/Snap--Build-Your-Own-Blocks#1777]
We want to be able to teach how to use error reports, but currently, we can't get a script pic of one being reported. Can we have that feature?

**Comment by cycomachead** [comment]
Are you trying to capture for a reporter, or a whole script?

IIRC, this is supposed to work for reporters.

**Comment by maryfries** [comment]
Yeah, it does work for reporters, but since commands can also report errors, those should work too, no?

**Comment by cycomachead** [comment]
Yeah, I just wanted to make sure error reporting still was working there. It didn't used to. 

I think the commands/scripts case isn't supported right now because this is tied to the "script pic with result" function, which initially wouldn't make sense on scripts. I do agree it's a useful thing to do.
